packages/www/components/StreamDetails/MultistreamTargetsTable/TargetStatusBadge.tsx

```diff
@@ -1,29 +1,46 @@
 import { MultistreamTarget, Stream } from "@livepeer.studio/api";
 
-import { MultistreamStatus } from "hooks/use-analyzer";
+import { Metrics, MultistreamStatus } from "hooks/use-analyzer";
 import { useMemo } from "react";
 import StatusBadge, { Variant as Status } from "../../StatusBadge";
+import moment from "moment";
 
 const computeStatus = (
   stream: Stream,
   target: MultistreamTarget,
   status: MultistreamStatus,
-  streamActiveSince: number | undefined
+  streamActiveSince: number | undefined,
+  metrics: Metrics
 ): Status => {
   if (
     status?.connected.lastProbeTime < streamActiveSince ||
     !streamActiveSince
   ) {
     status = null;
   }
-  const isConnected = status?.connected.status;
+
+  const currentTimestamp = moment().unix() * 1000;
+
+  const lastActive = metrics.MultistreamActiveSec.find(
+    (m) => m.dimensions.targetId === target.id
+  ).last[0];
+
+  const difference = (currentTimestamp - lastActive) / 1000;
+
+  const TIME_THRESHOLD = 30;
```

Comment:
For now this is fine, just a note for myself to make a double check with the update interval in the metrics